dataline-integrations/singer/base/dataline_entrypoint.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+import sys
+import enum
+
+
+class DatalineEntrypointType(enum.Enum):
```

Comment:
could we stick with java for this? for _just_ this script, the language choice feels trivial, but this is going to be the jumping of point for where we're doing to do field mapping and any other shims we need to do with singer. i'd rather be doing that in java.